main.py

```diff
@@ -89,8 +89,9 @@ async def signal_handler(sig):
         dummy_wallet1 = DummyWallet1()
         self.wallets[dummy_wallet1.get_identifier()] = dummy_wallet1
 
-        dummy_wallet2 = DummyWallet2()
-        self.wallets[dummy_wallet2.get_identifier()] = dummy_wallet2
+        iota_wallet = IotaTestnetWallet(os.path.join(options.statedir, 'sqlite'), None)
+        iota_wallet.create_wallet()
```

Comment:
Should the wallets be `create()`d in the main file? It is useful for debugging, but in the real world, the action could / should be taken from the REST API.

Reply:
I think this was allready on master before